doc/about_rendering.md

```diff
@@ -1,66 +0,0 @@
-# About Rendering
```

Comment:
Shouldn't we keep any of this?

Reply:
The first limitation (dynamic marker-width) is resolved with this (it's #64 ).

Regarding the second limitation (allow-overlap)... I think there is no point in saying this here. The limitation is still valid, but the problem resides in Tangram, and if tangram-reference it's ok, then the CartoCSSRenderer will throw an appropriate exception to fallback to raster.
There are many limitations, not only this one, I think this could be said in tangram-reference, but not here, particularly because this is Tangram's fault.
However, we could say here that texts aren't supported, since this is tangram-css fault, not Tangram's.

Reply:
Fair enough. I would keep the information about text/fonts.